bundlewatch.config.js

```diff
@@ -1,20 +1,21 @@
 // https://bundlewatch.io/#/getting-started/using-a-config-file
 
-const { readdirSync } = require('fs')
+const { readdirSync } = require('fs');
 
 const packageFolders = [
     'packages/components/atoms',
     'packages/components/molecules',
     'packages/components/organisms',
     'packages/services',
     'packages/tools'
-]
+];
 
 const excludedPackages = [
     'packages/services/f-metadata',
     'packages/tools/generator-component',
-    'packages/tools/storybook', 
-    'packages/services/f-wdio-utils'
+    'packages/tools/storybook',
+    'packages/services/f-wdio-utils',
+    'packages/components/organisms/f-status-banner' // Excluding as the footer is pushing this package over the 100kb.
```

Comment:
I've had to exclude this one. I've spend a bit of time trying to get the footer file down in size, need to look at it separately so have excluded this file rather than bumping the 100kb file for all packages. (This component sits at 136kb)

Reply:
Didn't see the PR for the status banner which included the footer, but would it be better to keep the footer out of the status banner? Do we need to include it?

Just thinking for things like registration and checkout we don't include the header as part of those components for instance – but if there's a reason we're doing it then happy for this to go through.

Reply:
Not sure if you meant to reolve this?  🙏 😆 

Fine either way, but just interested to know the reasoning

Reply:
Apologies I didn't see the comment you made yesterday :| So I cleaned this up earlier today and coreWeb now uses a default layout and a stripped back version of static page now: https://github.com/justeat/fozzie-components/pull/737

At the time I hadn't realised it was not required.

Reply:
👍  Was just a bit slow on the reviews while helping look after Alice! I've left a small comment on that PR too just to keep you on your toes 😆